CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Bug Fixes
 
 - Fixed a bug where if the first step(s) in a cycle are skipped then the cycle solution started from the model's initial conditions instead of from the last state of the previous cycle ([#3708](https://github.com/pybamm-team/PyBaMM/pull/3708))
+- Updated `plot_voltage_components.py` to support both `Simulation` and `Solution` objects. Added new methods in both `Simulation` and `Solution` classes for allow the syntax `simulation.plot_voltage_components` and `solution.plot_voltage_components`. Updated `test_plot_voltage_components.py` to reflect these changes ([#3723](https://github.com/pybamm-team/PyBaMM/pull/3723)).
```

Comment:
```suggestion
- Updated `plot_voltage_components.py` to support both `Simulation` and `Solution` objects. Added new methods in both `Simulation` and `Solution` classes for allow the syntax `simulation.plot_voltage_components` and `solution.plot_voltage_components`. Updated `test_plot_voltage_components.py` to reflect these changes ([#3723](https://github.com/pybamm-team/PyBaMM/pull/3723)).
- Fixed a bug where if the first step(s) in a cycle are skipped then the cycle solution started from the model's initial conditions instead of from the last state of the previous cycle ([#3708](https://github.com/pybamm-team/PyBaMM/pull/3708))
```

Small comment: for the CHANGELOG we sort contributions by decreasing PR number (i.e. newer PRs first).

Reply:
Sorry, I just corrected this oversight

Commit [here](https://github.com/pybamm-team/PyBaMM/pull/3723/commits/86cddb62fc1242bbac634f25c2a16ffb2e74393a)

Reply:
@AlessioBugetti Just as an FYI, you can accept suggestions too so you don't have to do a separate commit. You just need to press the "commit suggestion" button